public/js/user.js

```diff
@@ -246,7 +246,8 @@ apos.define('apostrophe-workflow', {
     self.enableForceExportWidget = function() {
       apos.ui.link('apos-workflow-force-export-widget', null, function($el) {
         var widgetId = $el.closest('[data-apos-widget-id]').attr('data-apos-widget-id');
-        var docId = $el.closest('[data-doc-id]').attr('data-doc-id');
+        // Skip up to the enclosing area with a real doc id, not a virtual or widget one
+        var docId = $el.closest('[data-doc-id^="c"]').attr('data-doc-id');
```

Comment:
What is this `c` matching? Would it be clearer to just be explicit about the full value?

Reply:
The c is present for a real credential module id such as will be present on any real doc in aposDocs. It's not present for a fake doc id on a virtual area or a newly added area. The lack of true doc ids on newly added areas is problematic, but this solution is correct even if that gets fixed in Apostrophe; it drills upwards until it finds a satisfactory doc id, which will happen in the outer area the area was added to.

So there's no "full value" to be explicit on, I really am looking for that "c".